filebeat/crawler/prospector.go

```diff
@@ -185,6 +185,10 @@ func (p *Prospector) setupHarvesterConfig() error {
 		return err
 	}
 
+	if config.MaxBytes == 0 {
+		config.MaxBytes = cfg.DefaultMaxBytes
+	}
```

Comment:
reminds me:
- what if max_bytes < 0 ?
- do we want to impose limit if max_bytes == 0?